components/ThemedDateTimePicker.tsx

```diff
@@ -37,6 +38,7 @@ const ThemedDateTimePicker: React.FC<ThemedDateTimePickerProps> = ({
         showTimeSelect
         dateFormat='MMMM d, yyyy h:mm aa'
         ref={ref}
+        minDate={new Date()}
```

Comment:
passed days are disabled, but past time is still available. Maybe need to write a custom filter https://stackoverflow.com/questions/59826534/react-datepicker-mintime-and-maxtime-not-works

Reply:
yes will add in later PR. Had a look at related github issues and it seems not trivial

Reply:
perfect